webroot/js/participantSummary.js

```diff
@@ -7,7 +7,7 @@ function participantSummaryErrorHandler(err, divContainer) {
 };
 
 function loadAllSitesCallback(result) {
-    siteList = JSON.parse(result).map((item) => { return { name: item.name, id: item.id } });
+    siteList = JSON.parse(result).map((item) => { return { name: item.name, id: item.id, clientTypes: item.clientTypes } });
```

Comment:
Still need this from the front end as I want to highlight the clientTypes in the UI.